src/Avalonia.Native/WindowImpl.cs

```diff
@@ -67,8 +67,6 @@ void IAvnWindowEvents.GotInputWhenDisabled()
             }
         }
 
-        public IAvnWindow Native => _native;
```

Comment:
Another property is IAvnWindowBase, which breaks CI build.
Maybe "new" keyword more suitable here?